modules/db.js

```diff
@@ -16,9 +16,7 @@ exports.init = function() {
         // if db file doesn't exist then create it
         try {
             log.debug(`Check that db exists: ${filePath}`);
-
-            fs.accessSync(filePath, fs.R_OK);
-
+            fs.accessSync(filePath, fs.W_OK);
```

Comment:
@hiddentao here i've changed to check if the file is writeable. we could circumvent permission errors more easily.